src/libLookup/Lookup.cpp

```diff
@@ -416,11 +416,6 @@ uint128_t Lookup::TryGettingResolvedIP(const Peer& peer) const {
 void Lookup::SendMessageToLookupNodes(const bytes& message) const {
   LOG_MARKER();
 
-  // LOG_GENERAL(INFO, "i am here " <<
-  // to_string(m_mediator.m_currentEpochNum).c_str())
-
-  // TODO: provide interface in P2PComm instead of repopulating the lookup into
```

Comment:
The idea was to just call `SendBroadcastMessage` directly on `m_lookupNodes`.  That was before we added more logic in the for loop below, so now it is not simply copying m_lookupNodes into another vector.  Also, we could not use `m_lookupNodes` directly anyway without having to hold on to `m_mutexLookupNodes` the whole time.  So, having a separate vector is still preferred.